symengine/ruby/callgrind_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+valgrind --tool=callgrind --dump-instr=yes --simulate-cache=yes --collect-jumps=yes ruby -S rspec spec
```

Comment:
This was a substitute to automate tasks by `rake`. Shifting it to `bin` at the root will require writing a path name everytime I want to execute it. How about we shift it to the root of the gem dir. I am trying to make it run from other place, a `LoadError` shows up everytime. Or we can bring back rake, I know how to do it with `rake`.